ui/com/user-info.jsx

```diff
@@ -173,7 +189,7 @@ function sortFollowedFirst (a, b) {
 }
 
 export class UserInfoFollowers extends AutoRefreshingComponent {
-  computeState(props) {
+  computeState(props, callback) {
```

Comment:
Make sure all of the `computeState` methods use `callback()` instead of returning the state. Most of them dont.